src/services/GlobalRolesService.ts

```diff
@@ -51,12 +58,22 @@ const list = async (
   );
 };
 
+const remove = async (users: User[]): Promise<Response> => {
+  const payload: RemoveGlobalRolesRequestPayload = {
+    userIds: users.map((user) => user.id),
+  };
+
+  return httpUsersGlobalRoles.delete2<RemoveGlobalRolesResponsePayload>({
+    entity: payload,
+  });
+};
+
```

Comment:
For a split second I thought the newly added function that takes in multiple users made the other function that takes in one user redundant, until I noticed they have distinct responsibilities.

This is clean and looks good 👍